Cargo.toml

```diff
@@ -14,6 +14,9 @@ documentation = "https://iroh.computer/docs"
 # Sadly this also needs to be updated in .github/workflows/ci.yml
 rust-version = "1.67"
 
+[profile.release]
+strip = "symbols"
```

Comment:
should we really be doing this? cc @Arqu

Reply:
I added that in a bid to try to reduce the size of the static linked file, but it actually has no effect now that I'm testing it with a clearer head.

Took it out.